openbas-api/src/main/resources/application.properties

```
@@ -19,10 +19,63 @@ openbas.with-proxy=false
 #openbas.admin.password=ChangeMe
 #openbas.admin.token=ChangeMe
 
-#Feature Flag
-openbas.disabled-dev-features=
+########################
+# RUNTIME DEPENDENCIES #
+########################
+# PostgreSQL, MinIO and RabbitMQ must be configured (and running) for OpenBAS to work properly
+
+### POSTGRESQL settings
+### see also: https://docs.openbas.io/latest/deployment/configuration/#postgresql
+# connection URL to the PostgreSQL server
+spring.datasource.url=jdbc:postgresql://...
```

*thought :* I'm wondering if we shouldn't make this field blank and give an example in the comment above as I've seen people leaving this field as is

---

Or maybe to be in line with other places in this .properties (but I'm afraid it might confuse even more users) 
```suggestion
spring.datasource.url=<jdbc:postgresql://...>
```

---

I agree, we can let a placeholder like <something>

---

See: [`050462a`](https://github.com/OpenBAS-Platform/openbas/pull/2354/commits/050462aa120d4ba40418a7e4de3c5653da6d241c)